mne/evoked.py

```diff
@@ -759,7 +759,11 @@ def __sub__(self, evoked):
         this_evoked = deepcopy(evoked)
         this_evoked.data *= -1.
         out = combine_evoked([self, this_evoked])
-        out.comment = self.comment + " - " + this_evoked.comment
+        if self.comment is None or this_evoked.comment is None:
+            warnings.warn('evoked.comment expects a string but is None')
```

Comment:
Are there normal circumstances under which `self.comment == None`? If so, then I don't think we need a warning here.